### DISCUSSION TITLE
openSource/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#784] [Announcements]
# openSource/

Hello 算法 (https://github.com/krahets/hello-algo) 介绍：《Hello 算法》：动画图解、一键运行的数据结构与算法教程，支持 Java, C++, Python, Go, JS, TS, C#, Swift, Rust, Dart, Zig 等语言。 本项目旨在打造一本开源免费、新手友好的数据结构与算法入门...

https://fearless-coding.github.io/openSource/

**Comment by ww-coding** [comment]
111
